### PR TITLE
[Security] Add attributes on Passport

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes 
| Deprecations? | no 
| Tickets       | n/a 
| License       | MIT
| Doc PR        | not yet

see https://github.com/symfonycorp/connect/pull/95
/cc @wouterj

### CODE REVIEW

File: src/Symfony/Component/Security/Http/Authenticator/Passport/Passport.php [hunk 1]
```diff
@@ -47,4 +49,22 @@ public function getUser(): UserInterface
...
+    }
+
+    /**
+     * @param mixed $default
+     *
+     * @return mixed
+     */
+    public function getAttribute(string $name, $default = null)
+    {
+        return $this->attributes[$name] ?: $default;
```
Comment (wouterj) [inline src/Symfony/Component/Security/Http/Authenticator/Passport/Passport.php:68]: This must be `??`, right?

**Review by wouterj — APPROVED** [review]
Thanks!

**Comment by wouterj** [comment]
Oh btw (I always forget that): should this be mentioned in the CHANGELOG?

**Comment by fabpot** [comment]
Done in 9cff4e1188

**Pull request merged**